src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/UserManagementServiceTests.cs

```diff
@@ -565,408 +808,217 @@ public async Task InviteUser_UserAlreadyExists()
-        var actionResult = result.AssertLeft();
-        actionResult.AssertValidationProblem(ValidationErrorMessages.InviteNotFound);
-    }
+            var userReleaseRoleRepository = new Mock<IUserReleaseRoleRepository>(Strict);
+            userReleaseRoleRepository
+                .Setup(mock => mock.RemoveForUser(internalUser.Id, It.IsAny<CancellationToken>()))
+                .Returns(Task.CompletedTask);
 
-    [Fact]
-    public async Task DeleteUser()
-    {
-        var internalUser = _dataFixture.DefaultUser().Generate();
+            var userPublicationInviteRepository = new Mock<IUserPublicationInviteRepository>(Strict);
+            userPublicationInviteRepository
+                .Setup(mock => mock.RemoveByUserEmail(internalUser.Email, It.IsAny<CancellationToken>()))
+                .Returns(Task.CompletedTask);
 
-        var identityUser = new ApplicationUser { Email = internalUser.Email };
+            var userPublicationRoleRepository = new Mock<IUserPublicationRoleRepository>(Strict);
+            userPublicationRoleRepository
+                .Setup(mock => mock.RemoveForUser(internalUser.Id, It.IsAny<CancellationToken>()))
+                .Returns(Task.CompletedTask);
 
-        var invite = new UserInvite { Email = internalUser.Email };
+            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
+            await using (var usersAndRolesDbContext = InMemoryUserAndRolesDbContext(usersAndRolesDbContextId))
+            {
+                var service = SetupUserManagementService(
+                    contentDbContext: contentDbContext,
+                    usersAndRolesDbContext: usersAndRolesDbContext,
+                    userManager: userManager.Object,
+                    userRepository: userRepository.Object,
+                    userReleaseInviteRepository: userReleaseInviteRepository.Object,
+                    userReleaseRoleRepository: userReleaseRoleRepository.Object,
+                    userPublicationInviteRepository: userPublicationInviteRepository.Object,
+                    userPublicationRoleRepository: userPublicationRoleRepository.Object
+                );
 
-        var contentDbContextId = Guid.NewGuid().ToString();
-        var usersAndRolesDbContextId = Guid.NewGuid().ToString();
+                var result = await service.DeleteUser(internalUser.Email);
+                result.AssertRight();
+            }
 
-        await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
-        {
-            contentDbContext.Users.Add(internalUser);
-            await contentDbContext.SaveChangesAsync();
+            VerifyAllMocks(
+                userManager,
+                userRepository,
+                userReleaseInviteRepository,
+                userReleaseRoleRepository,
+                userPublicationInviteRepository,
+                userPublicationRoleRepository
+            );
         }
```

Comment:
I think you can remove the `ContentDbContext` completely from this test now as it's no longer depending on setting up any data for it, and rely on the default context set up in the `SetupUserManagementService` method.